games-emulation/stella/stella-5.1.3.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit desktop gnome2-utils
+
+DESCRIPTION="Stella Atari 2600 VCS Emulator"
+HOMEPAGE="http://stella.sourceforge.net/"
```

Comment:
`HOMEPAGE` is outdated, please switch to current one.

Reply:
oops